NicolaeChristof/Descend-Into-Darkness
# Feature shovelpick

Added destructible walls and floors which will be the basis for the shovel pick. Simply walk up to a breakable mesh (currently bright red) and press E. I also went ahead and added some spicy bgm.

How to test:
- walk up to a breakable mesh and press E
- watch mesh collapse
- It should disappear after a few seconds
- put some headphones into your earballs and take a sniff. The new bgm is super spicy so make sure to bundle up nice and warm XD

Bugs:
- No known bugs
